src/tokenizer.js

```diff
@@ -102,16 +113,12 @@ function tokenize(input) {
     }
 
     throw new TypeError(
-        `Unexpected character: ${char} at ${current} in '${input}'`,
+      `Unexpected character: ${char} at ${current} in '${input}'`
     );
   }
 
   tokens.push(new Token(TokenTypes.EOF));
   return tokens;
 }
 
-export {
-    tokenize,
-    Token,
-    TokenTypes,
-};
+export { tokenize, Token, TokenTypes };
```

Comment:
Not that big of deal in this case, but avoid making style changes like this that aren't related to the addition.